### ISSUE TITLE
Throws error in node.js v18

### ISSUE THREAD

**Issue by avaly** [noppa/xmllint-wasm#7]
The library throws an error in node.js v18, but works fine in v14 and v16.

I ran the sample code from the readme together with two files from the test folder:

## Node v14

```
$ node -v && node ./xmllint-wasm.js 
v14.16.1
There were no errors!
undefined
```

## Node v16

```
$ node -v && node ./xmllint-wasm.js 
v16.5.0
There were no errors!
undefined
```

## Node v18

```
$ node -v && node ./xmllint-wasm.js 
v18.1.0
(node:14590) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
Unexpected error event from worker: TypeError: Failed to parse URL from /tmp/node_modules/xmllint-wasm/xmllint.wasm
TypeError [Error]: Failed to parse URL from /tmp/node_modules/xmllint-wasm/xmllint.wasm
    at new Request (node:internal/deps/undici/undici:4816:19)
    at Agent2.fetch2 (node:internal/deps/undici/undici:5544:29)
    ... 4 lines matching cause stack trace ...
    at /tmp/node_modules/xmllint-wasm/xmllint.js:65:149
    at Object.<anonymous> (/tmp/node_modules/xmllint-wasm/xmllint_worker.js:11:24)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10) {
  [cause]: TypeError [ERR_INVALID_URL]: Invalid URL
      at new NodeError (node:internal/errors:377:5)
      at URL.onParseError (node:internal/url:563:9)
      at new URL (node:internal/url:643:5)
      at new Request (node:internal/deps/undici/undici:4814:25)
      at Agent2.fetch2 (node:internal/deps/undici/undici:5544:29)
      at Object.fetch (node:internal/deps/undici/undici:6372:20)
      at fetch (node:internal/bootstrap/pre_execution:199:25)
      at /tmp/node_modules/xmllint-wasm/xmllint.js:64:424
      at /tmp/node_modules/xmllint-wasm/xmllint.js:65:126
      at /tmp/node_modules/xmllint-wasm/xmllint.js:65:149

```

**Comment by noppa** [comment]
This seems to be fixed in version 3.0.1

**Comment by noppa** [comment]
Closing as I believe this is fixed now. Feel free to reopen if it persists.
